ckanext/csrf_filter/anti_csrf.py

```diff
@@ -372,7 +372,7 @@ def apply_token(response, request=None):
         return response
 
     token = get_response_token(response)
-    html = insert_token(html, token)
+    html = insert_token(six.ensure_text(html), token)
     if hasattr(response, 'data'):
-        response.data = html
+        response.data = six.ensure_binary(html)
```

Comment:
The underlying `set_data` function takes either str or bytes, so this conversion isn't really needed... however the symmetry makes more sense to me than converting the `data` from bytes to str and then setting it back as str.